modules/nf-core/software/samtools/index.nf

```diff
@@ -0,0 +1,30 @@
+process SAMTOOLS_INDEX {
+   label 'cpus_8'
+
+    tag "${meta.id}"
+
+    publishDir params.outdir, mode: params.publish_dir_mode,
```

Comment:
```suggestion
       publishDir "${params.outdir}",
        mode: params.publish_dir_mode,
        saveAs: { filename -> saveFiles(filename:filename, options:options, publish_dir:getSoftwareName(task.process), publish_id:meta.id) }
```
This is how it is specified in the modules/samtools/index repo

Reply:
Need to look more on that one, as I need some very specific publish_dir settings